### DISCUSSION TITLE
Quasar axios baseUrl not updating

### DISCUSSION THREAD

**Discussion by lamseck-jonathan** [quasarframework/quasar#15014] [General - Components / Directives / etc]
I built a frontend app that i connected to local backend via axios baseUrl : 127.0.0.1..... 
Now that i deployed the backend i want to use the service deployed by setting a new url in axios (boot file)

import { boot } from 'quasar/wrappers';
import axios, { AxiosInstance } from 'axios';
import { baseUrl } from 'src/data/baseUrl'

declare module '@vue/runtime-core' {
  interface ComponentCustomProperties {
    $axios: AxiosInstance;
  }
}

// Be careful when using SSR for cross-request state pollution
// due to creating a Singleton instance here;
// If any client changes this (global) instance, it might be a
// good idea to move this instance creation inside of the
// "export default () => {}" function below (which runs individually
// for each client)
const api = axios.create({ baseURL: 'https://the-link-of-api/api' });

export default boot(({ app }) => {
  // for use inside Vue files (Options API) through this.$axios and this.$api

  app.config.globalProperties.$axios = axios;
  // ^ ^ ^ this will allow you to use this.$axios (for Vue Options API form)
  //       so you won't necessarily have to import axios in each vue file

  app.config.globalProperties.$api = api;
  // ^ ^ ^ this will allow you to use this.$api (for Vue Options API form)
  //       so you can easily perform requests against your app's API
});

export { api };

But each time i refresh my app , axios is still sending request to 127.0.0.1... and it is not working 
I am expecting the app to send request to the new baseUrl on axios boot file
What am i doing wrong ?

**Comment by lamseck-jonathan** [comment]
i have recently deployed my quasar app , so maybe when i run yarn quasar dev , it run the build folder instead of my actual project , when i make changes it is not applyed in local app

**Reply by metalsadman** [reply]
no idea what this means.

**Comment by metalsadman** [comment]
use the exported axios instance from your boot file, which is `api`, ie. `import { api } from 'boot/axios'`.
